### PR TITLE
feat: update docs to include steps on running old versions

### PR DESCRIPTION
**Pull request by wonderfulfrog**
This PR adds documentation steps for future users and maintainers on how to use older versions of Node.js and Python in order to run various Sweet Potato scripts.

### CODE REVIEW

File: documentation/pages/index.md [hunk 1]
```diff
@@ -19,7 +19,29 @@ The goal is to be simple and allow the composing and styling of pages without ge
...
+nvm install 15.14.0
+pyenv install 2.7.18
+```
+
+Running commands with older versions:
+
+```sh
+pyenv local 2.7.18
+nvm use 15.14.0
+nvm exec 15.14.0 <command>
```
Comment (andrewrubin) [inline documentation/pages/index.md]: Just curious, what's the reason behind the `nvm exec` line? Wouldn't `nvm use` switch to the correct Node version? What else is there to be done re: nvm?

**Review by andrewrubin — DISMISSED** [review]
This looks great, just one question from me, not a blocker

File: documentation/pages/index.md (hunk 1 above)
Comment (wonderfulfrog) [reply documentation/pages/index.md]: Oh I should probably elaborate in the steps here. Either option works, it's more of a user preference. I don't think Sweet Potato was set up for nvm yet so I provided both options.

Comment (wonderfulfrog) [reply documentation/pages/index.md]: Nevermind, it works fine even without `.nvmrc` files, so I'm going to drop the `npm exec` line.

**Review by andrewrubin — APPROVED** [review]
🚢

**Pull request merged**